conjur/api.py

```diff
@@ -285,6 +284,46 @@ def update_policy_file(self, policy_id, policy_file):
 
         return self._load_policy_file(policy_id, policy_file, HttpVerb.PATCH)
 
+    def rotate_anothers_api_key(self, resource, resource_to_rotate):
+        """
+        This method is used to rotate a user/host's API key
+        """
+        # Attach the resource type (user or host)
+        full_resource_id=':'.join([resource, resource_to_rotate])
```

Comment:
do you validate resource is valid? that is, user/host?

Reply:
I don't because resource comes from the CLI which is args.action (user or host). Now that I think about it, we are also the SDK 🙃  which doesnt have this information. So I will validate! Adding:
```
if resource != 'user' and resource != 'host':
     raise Exception("Error: Invalid resource type")
```

Reply:
You can do it using `in` operator in python, makes it more readable.